### ISSUE TITLE
ERROR: corrupted clipboard data, expected size=28 actual size=0

### ISSUE THREAD

**Issue by IGitYou** [symless/synergy#5534]
After upgrade to the latest stable build, I am no longer able to copy images from Win10 to Win10 using the x64 client, Frequently (if not persistently) the file errors in the logs with teh following

[2016-08-11T11:55:50] ERROR: corrupted clipboard data, expected size=1334452 actual size=69632
[2016-08-11T11:55:50] ERROR: corrupted clipboard data, expected size=1334452 actual size=0

This worked for a long time... although sometimes broke with the locked symlink (which this build is supposed to fix).

I searched issues and did not see it posted yet. Wondering if anyone else is seeing this or if it is a "me centric" issue.

Version 1.8.2-stable-36cd521

**Comment by mal** [comment]
Also seeing this:

Server: Windows 10
Client: Ubuntu 16.04
Version: 1.8.2-stable-36cd521

Nothing of interest in the server log, but corruption message in the client log. This is one example:

1. Copy URL on Windows server
2. Move cursor (once) to Ubuntu client
3. See this in client log, note that this was all from one screen entrance:

    ```
    [2016-08-13T21:07:07] INFO: entering screen
    [2016-08-13T21:07:07] INFO: leaving screen
    [2016-08-13T21:07:07] INFO: entering screen
    [2016-08-13T21:07:07] ERROR: corrupted clipboard data, expected size=94 actual size=0
    [2016-08-13T21:07:07] ERROR: corrupted clipboard data, expected size=94 actual size=0
    ```

**Comment by vanowm** [comment]
Can you confirm that when corrupted clipboard data occur, nothing is being copied into clipboard on the server?
That's what I'm suffering from, which is very frustrating when clipboard doesn't work on server itself, even when you are not planning transfer anything to the client.

P.S.
Are you saying you can copy files from server to client???

**Comment by mal** [comment]
Wow.

I'd previously missed that you were talking specifically about images. To test I took a screenshot on the server (Windows 10, Print Scrn) then moved the mouse over to the client (Ubuntu 16.04). The mouse flickered for a very short time, then centered itself on the client (as the client stoped/crashed).

The mouse pointer never visibly returned to the server, instead the CPU fan went nuts and all input was blocked. Ctrl+Alt+Delete was the only thing to function (not even Ctrl+Shift+Esc), from there I was able to start task manager (input appeared fine on the CAD menu) but could not interact with it as input was still hijacked, so ulimately had to use the CAD menu to force a reboot.

Obviously the server log was lost in the process, however I was able to retrieve the client log from the syslog:

```
Aug 14 12:06:45 hostname synergyc[13824]: [2016-08-14T12:06:45] INFO: entering screen
Aug 14 12:06:58 hostname synergyc[13824]: [2016-08-14T12:06:58] INFO: leaving screen
Aug 14 12:14:36 hostname synergyc[13824]: [2016-08-14T12:14:36] INFO: entering screen
Aug 14 12:14:36 hostname synergyc[13824]: [2016-08-14T12:14:36] INFO: clipboard was updated
Aug 14 12:14:36 hostname synergyc[13824]: [2016-08-14T12:14:36] INFO: clipboard was updated
Aug 14 12:14:37 hostname synergyc[13824]: [2016-08-14T12:14:37] INFO: leaving screen
Aug 14 12:14:42 hostname synergyc[13824]: [2016-08-14T12:14:42] INFO: entering screen
Aug 14 12:14:42 hostname synergyc[13824]: [2016-08-14T12:14:42] INFO: leaving screen
Aug 14 12:14:49 hostname synergyc[13824]: [2016-08-14T12:14:49] NOTE: server is dead
Aug 14 12:14:50 hostname synergyc[13824]: [2016-08-14T12:14:50] WARNING: failed to connect to server: server is not responding
Aug 14 12:14:51 hostname synergyc[13824]: [2016-08-14T12:14:51] NOTE: connecting to 'xxx.xxx.xxx.xxx': xxx.xxx.xxx.xxx:24800
Aug 14 12:14:51 hostname synergyc[13824]: [2016-08-14T12:14:51] INFO: OpenSSL 1.0.2g-fips  1 Mar 2016
Aug 14 12:15:02 hostname synergyc[13824]: [2016-08-14T12:15:02] ERROR: failed to connect secure socket
Aug 14 12:15:03 hostname synergyc[13824]: [2016-08-14T12:15:03] NOTE: disconnected from server
Aug 14 12:15:03 hostname synergyc[13824]: [2016-08-14T12:15:03] NOTE: stopped client
Aug 14 12:17:02 hostname systemd[1]: Starting Synergy client service...
Aug 14 12:17:02 hostname systemd[1]: Started Synergy client service.
Aug 14 12:17:02 hostname synergyc[26878]: [2016-08-14T12:17:02] NOTE: started client
```

Won't be trying to copy any images again soon :fearful:

**Comment by IGitYou** [comment]
For me it is not just images. Text too. I dont think my server is crashing either. Restarting the client has been bringing it back... but it seems I get 1ish copy and paste before it breaks.

**Comment by XinyuHou** [comment]
Merged into #5502
